### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Berdenson**
Generally cleaning up robot code

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -48,6 +48,7 @@ public class RobotContainer {
   /** mailboxBelts object for the MailboxBelts class */
   public static MailboxBelts mailboxBelts = new MailboxBelts();
 
+  /** intake object for the Intake class */
```
Comment (willitcode) [inline src/main/java/frc/robot/RobotContainer.java:51]: My dude, this is not descriptive. I would do something like the JavaDoc that we have for `RobotContainer.drive`

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -56,12 +57,12 @@ public class RobotContainer {
    * ************
    */
 
-  public static DriveRobotOriented driveRobotOriented = new DriveRobotOriented();
-  public static DriveFieldOriented driveFieldOriented = new DriveFieldOriented();
-  public static EnterXMode enterXMode = new EnterXMode();
-  public static DeployPneumatics deployPneumatics = new DeployPneumatics();
-  public static RunBelts runBelts = new RunBelts();
-  public static RunIntake runIntake = new RunIntake();
+  private static DriveRobotOriented driveRobotOriented = new DriveRobotOriented();
```
Comment (willitcode) [inline src/main/java/frc/robot/RobotContainer.java:60]: Make these not static, having them static is gonna super break stuff. I'm surprised this even compiled.

File: src/main/java/frc/robot/commands/DeployMailbox.java [hunk 3]
```diff
@@ -13,8 +13,9 @@
 
 import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
 
+/** Deploys the mailbox we use to output gamepieces. */
```
Comment (willitcode) [inline src/main/java/frc/robot/commands/DeployMailbox.java:16]: I'd add something about also running belts here

File: src/test/BaseTest.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ * Asimov's Laws:
+ * The First Law: A robot may not injure a human being or, through inaction, allow a human being to come to harm.
+ * The Second Law: A robot must obey the orders given it by human beings except where such orders would conflict with the First Law.
+ * The Third Law: A robot must protect its own existence as long as such protection does not conflict with the First or Second Law.
+ */
+
+import org.junit.jupiter.api.Test;
+
+public class BaseTests {
+  @Test
```
Comment (willitcode) [inline src/test/BaseTest.java:15]: Get rid of this test. We don't need a test that literally just prints "Test :)"

**Review by willitcode — CHANGES_REQUESTED** [review]

**Review by willitcode — APPROVED** [review]

**Comment by Berdenson** [comment]
https://youtu.be/hmIcv0FeJhs

**Pull request merged**